buildscripts/cmake/DeclareModuleSetup.cmake

```diff
@@ -117,31 +117,22 @@ macro(setup_module)
 
     if (BUILD_SHARED_LIBS)
         install(TARGETS ${MODULE} DESTINATION ${SHARED_LIBS_INSTALL_DESTINATION})
-
-        if (NOT MSVC)
-            set_target_properties(${MODULE} PROPERTIES COMPILE_FLAGS "-fPIC")
```

Comment:
It's on by default for shared libraries: https://cmake.org/cmake/help/latest/prop_tgt/POSITION_INDEPENDENT_CODE.html#prop_tgt:POSITION_INDEPENDENT_CODE
> This property is True by default for SHARED and MODULE library targets and False otherwise.